### PR TITLE
3.0: Disable ubuntu automatic update

### PR DESCRIPTION
**Pull request by yuleiwan**
APT::Periodic::Update-Package-Lists // Do "apt-get update" automatically every n-days (0=disable)
APT::Periodic::Unattended-Upgrade // Run the "unattended-upgrade" security upgrade script every n-days (0=disabled)

disable automatic update on ubuntu, convert
```
APT::Periodic::Update-Package-Lists "1";
APT::Periodic::Unattended-Upgrade "1";
```
to
```
APT::Periodic::Update-Package-Lists "0";
APT::Periodic::Unattended-Upgrade "0";
```

Signed-off-by: Yulei Wang <yuleiwan@amazon.com>

**Please See** [Git Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions)

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml [hunk 1]
```diff
@@ -204,6 +204,7 @@ phases:
                 apt-get clean
                 apt-get -y update
                 apt-get -y install build-essential curl wget jq
+                sed -i "s/1/0/g" /etc/apt/apt.conf.d/20auto-upgrades
```
Comment (lukeseawalker) [inline cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml:207]: Is it possible to be more strict on matching, something like
```
"s/Unattended-Upgrade \"1\"/Unattended-Upgrade \"0\"/g"
```

Comment (lukeseawalker) [reply cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml:207]: I see now that we are also changing `Update-Package-Lists`.

Comment (lukeseawalker) [reply cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml:207]: I would also move this change right after the apt lock check, right after the `flock` line

Comment (yuleiwan) [reply cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml]: Shall we move this change under condition ${CfnParamUpdateOsAndReboot} == false? Don't we disable ubuntu automatic update in all the cases?

**Review by lukeseawalker — APPROVED** [review]

**Pull request merged**